src/blocks/food-and-drinks/food-item/styles/style.scss

```diff
@@ -1,9 +1,25 @@
+
+:-ms-fullscreen .wp-block-coblocks-food-item,
+:root .wp-block-coblocks-food-item {
```

Comment:
I updated the selectors so we're only targeting our own block rather than using selectors that only exist within a theme. These selectors are a sweet add because it seems to just target IE like we want.